addon/components/table-column.js

```diff
@@ -118,6 +126,10 @@ export default Component.extend({
     if (this.get('groupWithPriorRow')) {
       run.scheduleOnce('sync', this, this._groupWithPriorRow);
     }
+
+    if (this.get('grayedOutRepeatedCells')) {
+      run.scheduleOnce('sync', this, this._grayedOutCellGroup);
+    }
```

Comment:
we probably don't want to allow both `groupWithPriorRow` and `grayedOutRepeatedCells`